toponetx/datasets/graph.py

```diff
@@ -0,0 +1,92 @@
+"""Various examples of named graphs represented as complexes."""
+
+import networkx as nx
+import numpy as np
+
+from toponetx import CellComplex
+from toponetx.algorithms.spectrum import (
+    hodge_laplacian_eigenvectors,
+    set_hodge_laplacian_eigenvector_attrs,
+)
+from toponetx.transform.graph_to_simplicial_complex import graph_2_clique_complex
+
+__all__ = ["karate_club"]
+
+
+def karate_club(complex_type="simplicial", feat_dim=2):
+    """Load the karate club as complex.
+
+        simplicial complex is returned as the clique complex of the graph
+        cell complex is return as the cell complex obtained by adding the independent cycles of graph
+
+    Parameters
+    ----------
+    complex_type : str, optional
+        The type of complex to load. Supported values are
+        "simplicial complex" and "cell complex".
```

Comment:
```suggestion
        "simplicial" and "cell".
```

Reply:
In general the correct way would be to note the possible values as literal types, not as part of the parameter description. E.g.:
```
complex_type : {'simplicial', 'cell'}, default='simplicial'
    The type of complex to load.
```
In addition, `default='simplex'` is more appropriate than `optional`.